client/src/redux/reducers/twitterReducer.js

```diff
@@ -36,6 +39,7 @@ const twitterReducer = (state = initialState, action) => {
         profileImage: '',
         isConnected: false,
         streams: [],
+        streamsById: {},
```

Comment:
Since this is a reset, we could use the initialState here.

Reply:
fixed